source/administration/object-management.rst

```diff
@@ -83,23 +102,31 @@ MinIO supports both :ref:`duration based object retention <minio-object-locking-
    :align: center
    :width: 600px
 
-See :ref:`minio-object-locking` for more complete documentation.
+Delete operations against a WORM-locked object depend on the specific operation:
+
+- Delete operations which do not specify a version ID result in the creation of a "Delete Marker"
+- Delete operations which specify the version ID of a locked object result in an locking error
+
+You *must* enable object locking during bucket creation.
+Enabling bucket creation also enables :ref:`versioning <minio-bucket-versioning>`.
 
 MinIO Object Locking provides key data retention compliance and meets SEC17a-4(f), FINRA 4511(C), and CFTC 1.31(c)-(d) requirements as per `Cohasset Associates <https://min.io/cohasset?ref=docs>`__.
 
+See :ref:`minio-object-locking` for more complete documentation.
+
 Object Lifecycle Management
 ---------------------------
 
-MinIO Object Lifecycle Management allows creating rules for time or date
-based automatic transition or expiry of objects. For object transition,
-MinIO automatically moves the object to a configured remote storage
-tier. For object expiry, MinIO automatically deletes the object.
+MinIO Object Lifecycle Management allows creating rules for time or date based automatic transition or expiry of objects. 
+For object transition, MinIO automatically moves the object to a configured remote storage tier. 
+For object expiry, MinIO automatically deletes the object.
+
+MinIO applies lifecycle management rules on :ref:`versioned and unversioned buckets <minio-bucket-versioning>` using the same behavior as normal client operations.
+You can specify transition or lifecycle rules that handle either or both the latest object versions and prior object versions.
```

Comment:
"or both" here feels oddly placed. Maybe "rules that handle the latest object versions, prior object versions, or both" ?

Reply:
I'll take that suggestion